core/src/main/resources/hudson/cli/Messages_de.properties

```diff
@@ -1,11 +1,64 @@
+InstallToolCommand.ShortDescription=Installiert ein Hilfsprogramm und gibt das Installationsverzeichnis aus. Dieser Befehl funktioniert nur innerhalb eines Builds.
+ListChangesCommand.ShortDescription=Zeigt das Changelog des angegebenen Builds.
+ListJobsCommand.ShortDescription=Zeigt alle Elemente, die zu einer Ansicht oder einem Ordner geh\u00F6ren.
+ListPluginsCommand.ShortDescription=Gibt eine Liste installierter Plugins aus.
+LoginCommand.ShortDescription=Speichert die aktuellen Anmeldedaten, damit weitere Befehle ohne gesonderte Anmeldung ausgef\u00FChrt werden k\u00F6nnen.
+LogoutCommand.ShortDescription=L\u00F6scht die zuvor mit login-Befehl gespeicherten Anmeldedaten.
+MailCommand.ShortDescription=Liest eine Nachricht von der Standardeingabe (stdin) und versendet sie als Email
+OfflineNodeCommand.ShortDescription=Knoten wird bis zum n\u00E4chsten "online-node"-Kommando f\u00FCr keine neuen Builds verwendet.
+OnlineNodeCommand.ShortDescription=Knoten wird wieder f\u00FCr neue Builds verwendet. Hebt ein vorausgegangenes "offline-node"-Kommando auf.
+QuietDownCommand.ShortDescription=Keine neuen Builds mehr starten, z.B. zur Vorbereitung eines Neustarts.
+ReloadConfigurationCommand.ShortDescription=Alle Daten im Speicher verwerfen und Konfiguration neu von Festplatte laden. Dies ist n\u00FCtzlich, wenn Sie \u00C4nderungen direkt im Dateisystem vorgenommen haben.
+ReloadJobCommand.ShortDescription=L\u00E4dt ein Element neu.
+RemoveJobFromViewCommand.ShortDescription=Entfernt Elemente aus einer Ansicht
+SessionIdCommand.ShortDescription=Gibt die Session ID aus, die sich bei jedem Neustart von Jenkins \u00E4ndert.
+SetBuildDescriptionCommand.ShortDescription=Setzt die Beschreibung eines Builds.
+SetBuildDisplayNameCommand.ShortDescription=Setzt den Anzeigenamen eines Builds.
+SetBuildParameterCommand.ShortDescription=\u00C4ndert die Parameter des aktuellen Builds.
+# TODO  Dieser Befehl funktioniert nur innerhalb eines Builds.?
+
+SetBuildResultCommand.ShortDescription=Setzt das Ergebnis des aktuellen Builds. Dieser Befehl funktioniert nur innerhalb eines Builds.
+UpdateJobCommand.ShortDescription=Aktualisiert die Konfiguration eines Elements basierend auf der via Standardeingabe (stdin) bereitgestellten XML-Repr\u00E4sentation. Das Gegenst\u00FCck zum Befehl get-job.
+UpdateNodeCommand.ShortDescription=Aktualisiert die Konfiguration eines Knotens basierend auf der via Standardeingabe (stdin) bereitgestellten XML-Repr\u00E4sentation. Das Gegenst\u00FCck zum Befehl get-node.
+UpdateViewCommand.ShortDescription=Aktualisiert die Konfiguration einer Ansicht basierend auf der via Standardeingabe (stdin) bereitgestellten XML-Repr\u00E4sentation. Das Gegenst\u00FCck zum Befehl get-view.
+VersionCommand.ShortDescription=Zeigt die aktuelle Version an.
+WaitNodeOfflineCommand.ShortDescription=Wartet bis ein Knoten nicht mehr f\u00FCr neue Builds verwendet werden kann.
+# TODO temporarily offline?
```

Comment:
Unclear to me whether this sets the temporarilyOffline flag or uses a different mechanism, something that should be incorporated into the description once clear.